avalon/tools/creator/app.py

```diff
@@ -227,8 +227,19 @@ def _on_data_changed(self):
                 subset_name = subset_name[0].upper() + subset_name[1:]
             result.setText("{}{}".format(family, subset_name))
 
+            # Indicate subset existence
+            if not subset_name:
+                subset.setStyleSheet("")
+                message = "Empty subset name .."
+            elif subset_name in existed_subsets:
+                subset.setStyleSheet("border-color: #4E76BB;")
+                message = "Existed subset, appending next version."
```

Comment:
This should probably be "Existing subset, appending next version".

Reply:
Solved !